### DISCUSSION TITLE
Print out appears to be blank

### DISCUSSION THREAD

**Discussion by jpcyang** [ONLYOFFICE/build_tools#441] [Q&A] [answered]
Hello, I just recently compiled v7.0.1 tag v7.0.1.11-1-g5b54f20 and start testing.  I noticed when I tried to print, no characters will show up in the preview and the final printout.  Could you check if you are also observing this issue?  Thank you!
<img width="1162" alt="Screen Shot 2022-02-23 at 9 03 22 PM" src="https://user-images.githubusercontent.com/6244441/155461318-94580a05-0f24-4c1b-9fbc-15290ea054a2.png">
<img width="1085" alt="Screen Shot 2022-02-23 at 9 03 44 PM" src="https://user-images.githubusercontent.com/6244441/155461321-4f2d8acf-38f0-4cab-8058-c18c3ded1cc0.png">
!

**Comment by ShockwaveNN** [comment, marked as answer]
I've checked compilied version and for it work fine:
![image](https://user-images.githubusercontent.com/668524/155507564-b63e0e69-3123-4ad5-9642-04e264cdaf78.png)


Used 7.0.1 sources

Maybe you got some errors in logs, or maybe there is some specific problems with some fonts
